www/src/components/homepage/homepage-newsletter.js

```diff
@@ -1,110 +1,98 @@
+  bg: `newsletter.background`,
+  border: t => `1px solid ${t.colors.newsletter.border}`,
+  borderRadius: 2,
+  flexDirection: `column`,
+  mb: 8,
+  p: 9,
+  pb: t => `calc(${t.space[9]} + ${t.space[stripedBorderHeight]})`,
+  position: `relative`,
 
-  :after {
-    border-radius: 0 0 ${p => p.theme.radii[2]} ${p => p.theme.radii[2]};
-    background: ${p => p.theme.colors.newsletter.background}
+  ":after": {
+    borderBottomLeftRadius: 2,
+    borderBottomRightRadius: 2,
+    background: t => `${t.colors.newsletter.background}
       repeating-linear-gradient(
         135deg,
-        ${p => p.theme.colors.newsletter.stripeColorA},
-        ${p => p.theme.colors.newsletter.stripeColorA} 20px,
+        ${t.colors.newsletter.stripeColorA},
+        ${t.colors.newsletter.stripeColorA} 20px,
         transparent 20px,
         transparent 40px,
-        ${p => p.theme.colors.newsletter.stripeColorB} 40px,
-        ${p => p.theme.colors.newsletter.stripeColorB} 60px,
+        ${t.colors.newsletter.stripeColorB} 40px,
+        ${t.colors.newsletter.stripeColorB} 60px,
         transparent 60px,
         transparent 80px
-      );
-    bottom: 0;
-    content: "";
-    height: ${p => p.theme.space[stripedBorderHeight]};
-    left: 0;
-    right: 0;
-    position: absolute;
-  }
+      )`,
+    bottom: 0,
+    content: `""`,
+    height: t => t.space[stripedBorderHeight],
+    left: 0,
+    right: 0,
+    position: `absolute`,
+  },
 
-  ${mediaQueries.lg} {
-    flex-direction: row;
-    justify-content: space-between;
+  [mediaQueries.lg]: {
+    flexDirection: `row`,
+    justifyContent: `space-between`,
 
-    > * {
-      flex-basis: 50%;
-    }
-  }
-`
+    "> *": {
+      flexBasis: `50%`,
+    },
+  },
+}
 
-const Ornament = styled(`span`)`
-  left: -${p => p.theme.space[1]};
-  position: absolute;
-  top: -${p => p.theme.space[2]};
-`
+const nameStyles = {
+  color: `textMuted`,
+  fontFamily: `heading`,
+  fontSize: 1,
+  fontWeight: `body`,
+  letterSpacing: `tracked`,
+  m: 0,
+  textTransform: `uppercase`,
+}
 
-const Name = styled(`h3`)`
-  color: ${p => p.theme.colors.textMuted};
-  font-family: ${p => p.theme.fonts.heading};
-  font-size: ${p => p.theme.fontSizes[1]};
-  font-weight: ${p => p.theme.fontWeights.body};
-  letter-spacing: ${p => p.theme.letterSpacings.tracked};
-  margin: 0;
-  text-transform: uppercase;
-`
-
-const Title = styled(`h1`)`
-  color: ${p => p.theme.colors.newsletter.heading};
-  font-size: ${p => p.theme.fontSizes[4]};
-  font-weight: ${p => p.theme.fontWeights.heading};
-  line-height: ${p => p.theme.lineHeights.dense};
-  margin: 0;
-  margin-top: ${p => p.theme.space[1]};
-`
-
-const Form = styled(EmailCaptureForm)`
-  margin-top: ${p => p.theme.space[5]};
-
-  ${mediaQueries.lg} {
-    margin-top: 0;
-  }
-`
+const titleStyles = {
+  color: `newsletter.heading`,
+  fontSize: 4,
+  fontWeight: `heading`,
+  lineHeight: `dense`,
+  m: 0,
+  mt: 1,
+}
 
 const HomepageNewsletter = () => (
```

Comment:
can we export it as a function declaration? we are doing it like this for all the other components which we are refactoring.

Reply:
👍